packages/dapp/src/services/LoanService.js

```diff
@@ -112,6 +112,26 @@ export function useBorrowedLoans(borrowerAddress: Address): Array<LoanData> {
   return loans
 }
 
+export function useBorrowedLoanById(
```

Comment:
Have you tested this function? Is it useful to retrieve a single loan, correct?

Reply:
Yes, it loads a single loan at a time. Its useful since we have always have the most updated version of the loan terms.